CharlieBackend.Api/CharlieBackend.Api.csproj

```diff
@@ -7,6 +7,7 @@
 
   <PropertyGroup>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <NoWarn>1591;</NoWarn>
```

Comment:
Adding warning _1591_ (_missing XML comment for publicly visible type or member_) to ignored so we won't get warnings like these
![ascascasc](https://user-images.githubusercontent.com/38729881/111506649-7f6f4a00-8752-11eb-9313-e5ea1d84c580.png)
both in IDE and in github. For example, here it gives these warning on class constructor and on `InvokeAsync()` method of the middleware class.

@mahotskuy 
Please, give us your opinion on this change.

Reply:
We've been using xml documentation the whole time. I think we have to generate a warning if XML documentation is missed.

Reply:
Removed warning 1591 from ignored in https://github.com/ita-social-projects/WhatBackend/pull/372/commits/79b57543b86ac6e84a925ffac5bdbe387fecaec6